scripts/tasks/src/swc/utils.ts

```diff
@@ -0,0 +1,29 @@
+import { type Options } from './types';
+
+const importPaths = {
+  es6: /from\s+["']([^"']+)["']/g,
+  commonjs: /require\(["']([^"']+)["']\)/g,
+};
+export function addJsExtensionToImports(code: string, type: Options['module']['type'] = 'es6') {
```

Comment:
not used anymore but keeping for now if we needed to do some extra codemods in future